pkg/kubernetes/mtls.go

```diff
@@ -107,6 +109,23 @@ func ExportTrustChain(outputDir string) error {
 	return nil
 }
 
+// Check and warn if cert expiry is less than 90 days.
+func WarnForCertExpiry() {
+	expiry, err := Expiry()
+	if err != nil {
+		print.FailureStatusEvent(os.Stderr, fmt.Sprintf("error getting root cert expiry: %s", err))
+		return
+	}
+	daysRemaining := int(expiry.Sub(time.Now().UTC()).Hours() / 24)
+	if daysRemaining < 30 {
```

Comment:
nit: Should we keep 30 as a constant above?

Reply:
no magic numbers please ... Put it as constants above in code ...